client/my-sites/post-share/style.scss

```diff
@@ -66,66 +65,28 @@ $section-border: solid 1px transparentize( lighten( $gray, 20% ), .5 );
 }
 
 .post-share__form {
-	flex: 3;
```

Comment:
The following changes into `.post-share__form{ ... }` could affect the aspect of the form at the sidebar of the post edition page?